example_cluster/README.md

```diff
@@ -0,0 +1,36 @@
+## Example Cluster
```

Comment:
These are nice docs. Why not move them into the docs folder so they end up on readthedocs and are easily referenced?